server/api.js

```diff
@@ -84,4 +85,88 @@ router.get("/quizes",(req, res)=>{
 	});
 });
 
+router.get("/quizes/search",(req, res)=>{
+	const client = getClient();
+	client.connect(()=>{
+		const db = client.db("quiz");
+		const collection = db.collection("quizes");
+		const quiz = { quiz_name:req.query.quiz_name };
+		collection.findOne(quiz,(err, questions)=>{
+			if(err){
+				res.status(500).send(err);
+			}else{
+				res.status(200).send(questions);
+			}
+		});
+	});
+});
+
+router.get("/results/search",(req, res)=>{
+	const client = getClient();
+	client.connect(()=>{
+		const db = client.db("quiz");
+		const collection = db.collection("results");
+		const quiz = { name:req.query.name };
```

Comment:
It's probably safer to have the search param be an `id` for a quiz (on the assumption that the frontend can pass that through so the user doesn't need to know it).

Otherwise we run into issues if we have multiple `Javascript Quiz 3` records in the database and we're trying to search by name which isn't unique.

Reply:
It's a great feedback, I'll do it as soon as possible.